src/containers/SortingUnitView.tsx

```diff
@@ -78,7 +78,7 @@ const SortingUnitView: FunctionComponent<Props> = ({ sortingId, unitId, sorting,
             calculationPool={calculationPool}
             width={width}
             sortingInfo={sortingInfo}
-            sortingUnitViews={sortingUnitViews}
+            plugins={plugins}
```

Comment:
Most of these changes relate to the change described in my above comment... renaming extensionContext redux state to plugins redux state, and passing plugins object rather than sortingViewPlugins, etc.